# Kinematic chain parameter
base_link: arm_base_link
tip_link: arm_7_link
